raw3-3-autobringup-kinetic/99-cob.rules

```diff
@@ -0,0 +1,48 @@
+#pcan usb devices
+#upper usb plug, always plug in pcan module for neck/tablet
+KERNELS=="2-2" SUBSYSTEM=="usb_device",ATTRS{idVendor}=="0c72",SYMLINK+="pcanneck"
+#lower usb plug, always plug in pcan module for arm
+KERNELS=="2-1" SUBSYSTEM=="usb_device",ATTRS{idVendor}=="0c72",SYMLINK+="pcangarm"
```

Comment:
`pcangarm` typo? where is the `g` coming from?

Reply:
The file was taken from `udev_rules/99-cob.rules` by @ipa-jba, I just fixed the LED device IDs since they are different for the raw3-3.

Reply:
I vote for fixing the type.

Reply:
I think most of these can be removed, I'll see if I can test which ones we need when the robot is back in biolab and I can install the kinetic PC.